Resources/Accessibility/Colors.md

```diff
@@ -3,12 +3,21 @@
 
 ## Why should you care?
 
+* Everybody uses the internet and each of them are different! Its unlikely that you will know every user so creating a website that some can't use is annoying to them and detrimental to your websites growth. Having more users helps expand the site's reach and increases the likelyhood of gaining more users in the future.
+
 ## Who does this affect?
 
+* The short answer is everybody! Ensuring that colors are easy to see and read is important for everyone but especially people who have color blindness or other impairments.
+
 ### What is color blindess?
 
 ## How to be proactive and work towards a more accesible web
 
 ## Common pitfalls
 
 ## Other resources
+
+* https://www.a11yproject.com/posts/what-is-color-contrast/
```

Comment:
Consider switching the order of the resource and its description. It may be more clear and intuitive for readers to read about a resource then have the link to that resource.  Or link the article and describe what it is like the [Resources/README External Resources section](https://github.com/PortfoliU-Project/PortfoliU-Project/tree/main/Resources#external-resources)

Reply:
Swapped the ordering to fit what you recommended.